services/web/server/src/simcore_service_webserver/redis.py

```diff
@@ -32,13 +32,16 @@ async def setup_redis_client(app: web.Application):
     """
     redis_settings: RedisSettings = get_plugin_settings(app)
     app[_APP_REDIS_CLIENTS_MANAGER] = manager = RedisClientsManager(
-        databases={
-            RedisDatabase.RESOURCES,
-            RedisDatabase.LOCKS,
-            RedisDatabase.VALIDATION_CODES,
-            RedisDatabase.SCHEDULED_MAINTENANCE,
-            RedisDatabase.USER_NOTIFICATIONS,
-            RedisDatabase.ANNOUNCEMENTS,
+        db_configs={
```

Comment:
TIP: when the name of the iter variable is unimportant you can use `_`. IMO it is more readable 
```python
db_configs={ RedisManagerDBConfig(_)
            for _ in (
                RedisDatabase.RESOURCES,
                RedisDatabase.LOCKS,
                RedisDatabase.VALIDATION_CODES,
                RedisDatabase.SCHEDULED_MAINTENANCE,
                RedisDatabase.USER_NOTIFICATIONS,
                RedisDatabase.ANNOUNCEMENTS,
            )
        }
```

Reply:
replied above why not to use `_`